cypress/integration/page.spec.js

```diff
@@ -64,509 +64,509 @@
+  //   it('should have correct address', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.assertHash('#/some/not/existing/page');
+  //   });
+
+  //   it('should have navigation', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.nav().should('exist');
+  //   });
+
+  //   it('should not have active nav links', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+
+  //     page.nav().contains('a', 'Home').should('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //     page.nav().contains('a', 'People').should('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.assetTitle('Page not found');
+  //   });
+
+  //   it('should not send API request', () => {
+  //     page.spyOnPeopleRequest();
+  //     page.visit('/#/some/not/existing/page');
+  //     cy.get('@peopleRequest').should('not.be.called');
+  //   });
+
+  //   it('should not have loader', () => {
+  //     cy.clock();
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.loader().should('not.exist');
+  //   });
+
+  //   it('should not have the `no people` message', () => {
+  //     page.mockNoPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.noPeopleMessage().should('not.exist');
+  //   });
+
+  //   it('should not have loading error', () => {
+  //     page.mockPeopleError();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.peopleLoadingError().should('not.exist');
+  //   });
+
+  //   it('should not have people table', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.peopleTable().should('not.exist');
+  //   });
+  // });
+
+  // describe('#/people page', () => {
+  //   it('should have correct address', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/people');
+  //     page.assertHash('#/people');
+  //   });
+
+  //   it('should have navigation', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/people');
+  //     page.nav().should('exist');
+  //   });
+
+  //   it('should have People nav link active', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/people');
+  //     page.nav()
+  //       .contains('a', 'People')
+  //       .and('have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have Home nav link not active', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/people');
+  //     page.nav()
+  //       .contains('a', 'Home')
+  //       .and('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/people');
+  //     page.assetTitle('People Page');
```

Comment:
The function `assetTitle` is likely a typo and should be `assertTitle` to correctly describe its purpose of asserting the page title.